### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rachelzhang0922**
updated the introducion based on discussion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,13 +18,13 @@
...
+***Shifu*** is a native framework to help IoT developers interact with physical devices. ***Shifu*** virtualizes the management and monitoring of devices, giving them the ability to run automatically. It solves the troubles of protocol and driver integration and enables devices to be plug-and-play.
 
 ## Why ***Shifu***
 
-***Shifu*** makes it extremely simple to manage and control your IoT devices via ***deviceShifu***, a digital twin for your device. When you connect to an IoT device, ***Shifu*** will recognize it and start a ***deviceShifu*** of the device as a [Kubernetes Pod](https://kubernetes.io/docs/concepts/workloads/pods/).
-
-***deviceShifu*** provides you with a high-level abstraction to interact with your device. By implementing the interface of the ***deviceShifu***, your IoT device can achieve everything it is designed for, and much more! For example, you can have your device actively push its telemetry to any endpoint of your choice.
+- ***Shifu*** is a Kubernetes native framework that achieves aerospace-grade stability (up to 99.9999%).
```
Comment (tomqin93) [inline README.md:25]: should we reference `https://kubernetes.io` for `Kubernetes`?
`aerospace-grade` => `aerospace grade`

File: README.md [hunk 2]
```diff
@@ -18,13 +18,13 @@
...
+- Device protocols and drivers are compiled as configuration files, which are efficient and reusable.
```
Comment (tomqin93) [inline README.md:26]: ... from configuration files,...

**Review by tomqin93 — COMMENTED** [review]
@rachelzhang0922 added 2 comments

File: README.md [hunk 3]
```diff
@@ -18,13 +18,13 @@
 
 </div>
 
-***Shifu*** is a [Kubernetes](https://kubernetes.io) native framework designed to abstract out the complexity of interacting with IoT devices. ***Shifu*** aims to achieve TRUE plug-and-play IoT device virtualization, management, control, and automation.
+***Shifu*** is a native framework to help IoT developers interact with physical devices. ***Shifu*** virtualizes the management and monitoring of devices, giving them the ability to run automatically. It solves the troubles of protocol and driver integration and enables devices to be plug-and-play.
```
Comment (BtXin) [inline README.md:21]: `native` => `kubernetes native` 
Native in computer science usually means it is written in C/C++

**Comment by BtXin** [comment]
I heard that we are trying to avoid using the word `kubernetes` in readme. In that case you can change `kubernetes native` to `cloud native`.

File: README.md [hunk 4]
```diff
@@ -18,13 +18,13 @@
...
 
 ## Why ***Shifu***
 
-***Shifu*** makes it extremely simple to manage and control your IoT devices via ***deviceShifu***, a digital twin for your device. When you connect to an IoT device, ***Shifu*** will recognize it and start a ***deviceShifu*** of the device as a [Kubernetes Pod](https://kubernetes.io/docs/concepts/workloads/pods/).
-
-***deviceShifu*** provides you with a high-level abstraction to interact with your device. By implementing the interface of the ***deviceShifu***, your IoT device can achieve everything it is designed for, and much more! For example, you can have your device actively push its telemetry to any endpoint of your choice.
+- ***Shifu*** is a cloud native framework that achieves aerospace level stability (up to 99.9999%).
+- Device protocols and drivers are compiled from configuration files, which are efficient and reusable.
+- Modular deployment of devices, developers can simply load them on demand.
```
Comment (BtXin) [inline README.md:27]: Modularized deployment

**Review by saiyan86 — CHANGES_REQUESTED** [review]

**Comment by saiyan86** [comment]
why not reuse the highlights on https://shifu.run/ ?

File: README.md [hunk 5]
```diff
@@ -18,13 +18,13 @@
...
+***Shifu*** is a cloud native framework to help IoT developers interact with physical devices. ***Shifu*** virtualizes the management and monitoring of devices, giving them the ability to run automatically. It solves the troubles of protocol and driver integration and enables devices to be plug-and-play.
 
 ## Why ***Shifu***
 
-***Shifu*** makes it extremely simple to manage and control your IoT devices via ***deviceShifu***, a digital twin for your device. When you connect to an IoT device, ***Shifu*** will recognize it and start a ***deviceShifu*** of the device as a [Kubernetes Pod](https://kubernetes.io/docs/concepts/workloads/pods/).
-
-***deviceShifu*** provides you with a high-level abstraction to interact with your device. By implementing the interface of the ***deviceShifu***, your IoT device can achieve everything it is designed for, and much more! For example, you can have your device actively push its telemetry to any endpoint of your choice.
+- ***Shifu*** is a cloud native framework that achieves aerospace level stability (up to 99.9999%).
+- Device protocols and drivers are compiled from configuration files, which are efficient and reusable.
```
[the remark this answers is not in the document]
Comment (rachelzhang0922) [reply README.md:26]: should we add it as the third item or replace the whole context?

**Pull request closed without merging**